views/content/cli.md

```diff
@@ -186,15 +186,15 @@ The console has auto completion and history support that persists between sessio
 
 #### Learn More on Running a node
 
-* [Backup and restore](https://ethereum-homestead.readthedocs.org/en/latest/using-ethereum-the-basics/backup-import-accounts.html)
-* [Connecting to the network](https://ethereum-homestead.readthedocs.org/en/latest/using-ethereum-the-basics/connecting-to-the-network.html)
+* [Backup and restore](https://ethereum.gitbooks.io/frontier-guide/content/backup_restore.html)
+* [Connecting to the network](https://ethereum.gitbooks.io/frontier-guide/content/connecting.html)
 
 
 ### Usage examples
 
 #### Creating accounts
 
-In order to do anything on an Ethereum network you need ether, and to get it, you will need to generate an account. There are [various ways to go around this](https://ethereum-homestead.readthedocs.org/en/latest/using-ethereum-the-basics/account-management.html), but the simplest one is through the console.
```

Comment:
These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.